README.md

```diff
@@ -8,7 +8,9 @@
 [![Conda][conda-badge]][conda-link]
 [![PyPI - Downloads][install-badge]][install-link]
 
-An extended commonmark compliant parser, with bridges to docutils & sphinx.
```

Comment:
could you move this back to all being on one line? We try to keep sentences on the same line so that they're easier to make edits to and diff in github